pre_vs_berkeley.md

```diff
@@ -0,0 +1,440 @@
+# Merging the Berkeley schema into the NMDC schema
+
+_Corresponding to a jump from major version 10 to 11_
+
+This major refactoring increases the number of **classes** in the schema from 70 to 75, but decreases the number of root
+classes from 19 to 17 due to better organization.
```

Comment:
When I first read this sentence, I wondered what a "root class" was. @turbomam explained to me via Zoom that it is a class that is not `is_a` anything.